src/lib/components/permissions/row.svelte

```diff
@@ -38,6 +38,8 @@
             <div>Guests</div>
         {:else if role === 'any'}
             <div>Any</div>
+        {:else if role.startsWith('user') || role.startsWith('team')}
+            <div>{role}</div>
```

Comment:
I think this would change the behavior of if you put a specific team/user, right? This wouldn't work anymore, right?

<img width="887" alt="image" src="https://github.com/appwrite/console/assets/1477010/93b983a2-a618-4e59-b325-0568b9400ab0">